modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-sales/index.js

```diff
@@ -98,31 +307,56 @@ export default function () {
 		);
 	};
 
+	const getDateUntilGoal = getData()[0]?.dateUntilGoal;
+	const getSalesValue = getData()[0]?.salesValue;
+	const getGoalValue = getData()[0]?.goalValue;
+	const getSalesPercentual =
+		getData()[0]?.salesPercentual > 100
+			? `${100}%`
+			: `${getData()[0]?.salesPercentual}%`;
+
+	const isGoalRached = getSalesValue >= getGoalValue;
+
+	const colors = {
+		reached: !isGoalRached ? '#ec0d6b' : '#FD7E14',
+		remaining: '#fbcee1',
+	};
+
 	const chartData = {
 		colors,
 		columns: getData()[0]?.dataColumns,
 		type: 'donut',
 	};
 
-	const getDateUntilGoal = getData()[0]?.dateUntilGoal;
-	const getSalesValue = getData()[0]?.salesValue;
-	const getGoalValue = getData()[0]?.goalValue;
-	const getSalesPercentual = getData()[0]?.salesPercentual;
-
 	const LegendElement = () => (
-		<div className="d-flex donut-chart-legend flex-column h-100 justify-content-end ml-5 mt-5">
+		<div className="d-flex donut-chart-space-legend flex-column h-100 justify-content-end mt-5">
 			<div className="donut-chart-screen font-weight-bolder h5">
```

Comment:
This class is specific to the donut chart component. Please check as if someone changes this class their code will be impacted

Reply:
Done.